sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryUtils.java

```diff
@@ -710,7 +708,7 @@ public static Row toBeamRow(Schema rowSchema, TableSchema bqSchema, TableRow jso
                       (!innerTypeIsMap && v instanceof Map)
                           ? ((Map<String, Object>) v).get("v")
                           : v)
-              .map(v -> toBeamValue(fieldType.getCollectionElementType(), v))
+              .map(v -> toBeamValue(field.withType(field.getType().getCollectionElementType()), v))
```

Comment:
```suggestion
              .map(v -> toBeamValue(field.withType(fieldType.getCollectionElementType()), v))
```

LGTM: just a minor place

Reply:
ah missed it, thanks!